src/integration/pages/articles/scotland/amp.test.js

```diff
@@ -3,12 +3,10 @@
  * @pathname /scotland/articles/czwj5l0n210o
  */
 
-import runCrossPlatformTests from '../crossPlatformTests';
 import runAmpTests from '../ampTests';
 
 describe('AMP', () => {
   describe(pageType, () => {
-    runCrossPlatformTests('scotland');
-    runAmpTests();
+    runAmpTests(service);
```

Comment:
Where does `service` come from here?

Reply:
By declaring the `@service` in the header, it is then converted into a global var using docblock pragmas
https://github.com/bbc/simorgh/blob/latest/src/integration/integrationTestEnvironment.js#L14

Reply:
https://www.npmjs.com/package/jest-docblock

Reply:
Nice, thank you! Just realised this was in the README 👍